vellum/workflows/triage_message/nodes/process_job_node.py

```diff
@@ -38,6 +38,23 @@ def mark_job_as_blocked(reason: str):
     pass
 
 
+def split_job(sub_jobs: list, repo: Optional[str] = None):
+    """
+    Split the current job into smaller sub-jobs when the job is too large or complex
+    to complete in a single session. Each sub-job will be tracked as a GitHub issue.
+    
+    Args:
+        sub_jobs: List of sub-job definitions, each containing:
+            - name: Short descriptive name for the sub-job
+            - description: Detailed description of what needs to be done
+            - priority: Priority level (1.0 is normal, higher is more urgent)
+        repo: Optional GitHub repository (owner/repo format) to create issues in.
+              Must be one of: dvargas92495/vargasjr-dev, Cari-AI/cari-ai.
```

Comment:
Remove this line and the next one. These are not the only two valid repos, VargasJR will have access to an org full of repos that it manages, and may also manage some repos in client orgs